CurrencyConverterControl/CurrencyConverterControl.cs

```diff
@@ -146,12 +143,20 @@ private static void OutputValue_Textchanged(DependencyObject d, DependencyProper
         {
             CurrencyConverterControl control = d as CurrencyConverterControl;
             if (control != null && CurrencyConverter is  null)
```

Comment:
I don't get why you check for CurrencyConverter for null here. Why don't fire event if it's not null?

Reply:
In my thoughts was that if CurrencyConverter is null it means that it hasn't entered into the onApplyTemplate of the controller, and the change which triggers OutputValue Change comes from another project.

Reply:
just removed that check.